rc/tools/spell.kak

```diff
@@ -49,14 +51,10 @@ define-command -params ..1 -docstring %{
                         # drop the identification message
                     }
 
-                    else if (/^\*/) {
+                    else if (/^[-*+]/) {
                         # nothing
                     }
 
-                    else if (/^\+/) {
-                        # required to ignore undocumented aspell functionality
```

Comment:
We lost the comment here, which documents that `+` is an undocumented control character.

Reply:
I just thought it does not matter if we ignore these because of aspell bug or because of hunspell. But You are right, we should not drop the comment. I think I will add the hunspell info too as that is also related info.